docs/lab/at-home-phlebotomy/communications.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Communications"
+---
+
+Communications with the patient are carried in two different ways for At-Home Phlebotomy orders, being them
+email and SMS messages. There are three distinct communications options:
+
+- `Default` - Email and SMS communications are enabled.
+- `SMS Only` - Only SMS communication is enabled.
+- `Disable` - All communications from Vital are disabled.
+
+Each of these has a different set of content and status changes that trigger them, described in the following sections.
```

Comment:
Each option has a different set of content and status changes, depending on what triggered them.